### ISSUE TITLE
Asset failed to deserialize

### ISSUE THREAD

**Issue by awsomespider** [nesrak1/UABEA#366]
I am using UABEAvalonia downloaded from quick downloads '[Latest Nightly (Windows)' to be specific. I used it to get into globalgamemanagers and into build settings, however whenever I export a dump file of it and import it as a dump file (even when it hasnt been unedited), it modifies it with the star as it should be. However when I try to view data it says "Asset failed to deserialize" with a message saying 'The file version may be too new for this tpk or the file format is custom'

tl;dr: I am trying to modify 'build settings' in global game managers but when i import even the exact same dump file it gives an error when viewing data saying 'Asset failed to deserialize' with a message 'The file version may be too new for this tpk or the file format is custom.'
